### PR TITLE
Add linter for reference files

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

**Comment by flodolo** [comment]
@maxxcrawford 

This should fix #1

You can see an example of a failed run here https://github.com/flodolo/fx-private-relay-l10n/runs/6305269262?check_suite_focus=true

File: en/app.ftl [hunk 1]
```diff
@@ -190,7 +190,6 @@ landing-pricing-body-2 = Try { -brand-name-firefox-relay } email masks and start
 landing-pricing-free-price = Free
 landing-pricing-free-feature-1-2 = Up to 5 email masks
 landing-pricing-free-feature-2 = Browser extension
-landing-pricing-free-feature-free = Free
```
Comment (flodolo) [inline en/app.ftl:193]: Note: this was duplicated on line 198

**Review by Vinnl — APPROVED** [review]
Woah, I didn't know this existed (I was of half a mind to write something like this myself :stuck_out_tongue: ) - this is awesome @flodolo!

**Comment by flodolo** [comment]
> Woah, I didn't know this existed (I was of half a mind to write something like this myself 😛 ) - this is awesome @flodolo!

Well, it didn't exist until yesterday (at least in a packaged form) ;-)

I'm going to merge this, so we can test it further with incoming PRs.

**Pull request merged**